gems/aws-sdk-core/lib/aws-sdk-core/plugins/protocols/rest_json.rb

```diff
@@ -5,7 +5,23 @@ module Plugins
     module Protocols
       class RestJson < Seahorse::Client::Plugin
 
+        class ContentTypeHandler < Seahorse::Client::Handler
+          def call(context)
+            if modeled_body?(context)
```

Comment:
This is currently wrong. I think it needs to be for payloads, with either the payload trait, or any members that aren't bound to URI or header.